### DISCUSSION TITLE
Cannot make my Private Rep to public

### DISCUSSION THREAD

**Discussion by ssgali** [community/community#123147] [New to GitHub]
https://github.com/code50/82104662
This is my private repository, there is no such setting showing for it to be set to public. 
I am new to GITHUB please help,
Thanks
![image](https://github.com/community/community/assets/82104662/c375a671-2494-41ed-8296-d21711c0720b)

**Comment by davevad93** [comment]
Hi @ssgali ,

To change your repository visibility from `private` to `public` follow the steps below:

1. Go to your repo and click on `Settings` (in the top bar)

![1](https://github.com/community/community/assets/94308234/d6f99642-05bd-453e-b62e-249abc06187b)

2. In the `General Tab` go to the bottom until you see the `Danger Zone` section.

3. Click on `Change visibility`.

![2](https://github.com/community/community/assets/94308234/57197e75-301f-4c8b-94af-4b053a49c417)

4. Set it to `public`

**Reply by ssgali** [reply]
Hi @davevad93 I appreciate your response to my query but as you can see in the attached image there is no such option showing.

**Reply by davevad93** [reply]
Hi, yes I just noticed, you don't get any option because that is not your current account, you have to login with `code50` account but now you are logged in with `ssgali`, that's the reason why you don't see anything. If that is another account of yours log in it and follow the steps in my previous answer.

**Reply by ssgali** [reply]
Ohh thanks a lot, I'll look into it.
